terraform/aws/README.md

```diff
@@ -168,7 +168,7 @@ Internally to the subnet, all traffic is allowed.
 
 By default, this configuration creates 3 instances in AWS: one for support services (mainly iSCSI as most other services - DHCP, NTP, etc - are provided by Amazon) and 2 cluster nodes, but this can be changed to deploy more cluster nodes as needed.
 
-Once the infrastructure is created by Terraform, the servers are provisioned with Ansible.
+Once the infrastructure is created by Terraform, the servers can be provisioned with Ansible.
```

Comment:
It is not something under Terraform control. It is not Terraform running Ansible (as it was at the time Ansible was Salt). This is the same reason that allow to delete all the unused terraform variables.